<launch>
    <remap from="image" to="/iris/usb_cam/image_raw"/>
    <remap from="camera_info" to="/iris/usb_cam/camera_info"/>

    <arg name="config_path" default="$(find mavlink_indoor_sdk_sitl)/config/config.yml"/>
    

    <node name="udp_vpe_sitl" pkg="mavlink_indoor_sdk_sitl" type="udp_vpe_sitl" output="screen" >
        <param name="config_path" type="string" value="$(arg config_path)" />
    </node>
</launch>